packages/osl-data-index-ephemeral-image/package.json

```
@@ -0,0 +1,50 @@
+{
+  "private": true,
+  "name": "@osl/osl-data-index-ephemeral-image",
+  "version": "0.0.0",
+  "description": "",
+  "license": "Apache-2.0",
+  "homepage": "https://github.com/kubesmarts/kie-tools",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/kubesmarts/kie-tools.git"
+  },
+  "bugs": {
+    "url": "https://issues.redhat.com/projects/SRVLOGIC"
+  },
+  "scripts": {
+    "build:dev": "pnpm cleanup && run-script-if --bool \"$(build-env containerImages.build)\" --then \"pnpm image:cekit:copy\" \"pnpm image:cekit:cache:artifacts\" \"pnpm image:cekit:generate\"",
+    "build:prod": "pnpm cleanup && pnpm build:dev",
+    "cleanup": "rimraf dist && mkdir -p dist",
+    "format": "prettier --write . --ignore-path=../../.prettierignore --ignore-path=../../.gitignore",
+    "image:cekit:build": "run-script-os",
+    "image:cekit:build:linux:darwin": "pnpm setup:env make -C ./dist build",
+    "image:cekit:build:win32": "echo \"Build skipped on Windows\"",
+    "image:cekit:cache:artifacts": ". ./node_modules/@osl/redhat-python-venv/venv/bin/activate && . ./dist/scripts/process-cekit-artifact.sh $(build-env oslDataIndexEphemeralImage.artifactUrl) modules/osl-data-index-ephemeral/module.yaml",
```

As we talked yesterday, we can skip exporting the `OSL_DATA_INDEX_EPHEMERAL_IMAGE__artifactUrl` as from productization side CPaaS is able to get the latest productized artifact on PNC and use it during the build.

It is correctly impleted in the `packages/osl-data-index-ephemeral-image/dist/scripts/process-cekit-artifact.sh` script where if `FILE_URL` is empty, the script exist with status 0.

The problem is that if in the invocation the variable is not defined, the script will receive the second parameter (which is the module path) in the position of the first, so I suggest that we force a quote, so the first parameter will be in fact empty in case the `OSL_DATA_INDEX_EPHEMERAL_IMAGE__artifactUrl` is not exported, something like:

```suggestion
    "image:cekit:cache:artifacts": ". ./node_modules/@osl/redhat-python-venv/venv/bin/activate && . ./dist/scripts/process-cekit-artifact.sh \"$(build-env oslDataIndexEphemeralImage.artifactUrl)\" modules/osl-data-index-ephemeral/module.yaml",
```